### ISSUE TITLE
What are neural networks? IBM

### ISSUE THREAD

**Issue by Jkoles28** [Jkoles28/CS373#3]
### Title
What is a Neural Network?
### URL
https://www.ibm.com/topics/neural-networks
### Summary 
Neural networks are a type of machine learning algorithm that are modeled on the way biological neurons signal to one another. They are comprised of an input layer, one or more hidden layers, and an output layer, with each node, or artificial neuron, connecting to another and having an associated weight and threshold. Neural networks rely on training data to learn and improve their accuracy over time and can be used for tasks like speech recognition and image recognition. Deep learning is a type of neural network that has multiple layers, while neural networks with two or three layers are just basic neural networks. 
### Key Points 
* Neural networks are a type of machine learning algorithm modeled after the structure of the human brain. 
* Neural networks are typically composed of input layers, hidden layers, and output layers. Each node, or artificial neuron, passes data from one layer to the next, with weights and thresholds to determine activation of the node. Training data is used to learn and improve accuracy over time. 
* Deep Learning is a type of neural network that consists of more than three layers(2+ hidden layers).
* Types of neural networks: perceptrons, feedforward neural networks, convolutional neural networks, and recurrent neural networks. 
* Neural networks have been studied since the 1940s, with key milestones including Warren S. McCulloch and Walter Pitts' research, Frank Rosenblatt's development of the perceptron, Paul Werbos' work on backpropagation, and Yann LeCun's use of constraints in backpropagation. (Background?)
### Citation
IBM, “What are Neural Networks? | IBM,” www.ibm.com. https://www.ibm.com/topics/neural-networks
